src/sass/formio.form.scss

```diff
@@ -1229,3 +1229,7 @@ div[read-only="true"] {
     padding-left: 12px;
   }
 }
+
+.pagination {
```

Comment:
Wouldn't this change ALL paginations on the page and not just the header?  I would think that you would get a little more selective with this CSS.

Reply:
Yep, my fault.